docs/shared/filter_topo.yaml

```
@@ -0,0 +1,27 @@
+filter_topo:
+  config:
+    input_grid_file: /path/to/C403_grid.tile7.halo6.nc
+  execution:
+    batchargs:
+      cores: 1
+      walltime: "00:01:00"
+    envcmds:
+      - module use /path/to/modules
+      - module load module_name
+    executable: /path/to/filter_topo
+  namelist:
+    update_values:
+      filter_topo_nml:
+        grid_file: /path/to/C{{ filter_topo.namelist.filter_topo_nml.res }}_mosaic.halo6.nc
```

```suggestion
        grid_file: /path/to/C{{ filter_topo.namelist["update_values"]["filter_topo_nml"]["res"] }}_mosaic.halo6.nc
```
I wish we could do dots all the way down! In my experience, only the first one works that way.

---

I just re-executed the example run referenced in the PR _Description_ with this all-dots syntax, and it seems fine to me. Do you have a counterexample?

---

Hmmm. I will revisit this later. Perhaps I'd assumed that was an issue when I'd had failures in the past, but it was something else?